Metadata and store assets (#318)
hyperledger/iroha-2-docs: src/guide/configure/metadata-and-store-assets.md

# Choosing between the Store assets and metadata

> The `Store` assets and `metadata` allow storing several parameters with different types and have string keys. Despite the similarity, their use cases differ.

Reviewer: ```suggestion
The `Store` and `metadata` assets allow for the storage of several parameters with different types and string keys. Despite the similarity, their use cases differ.
```

> You must use `metadata` when you don’t want sequences of records, for example, extending user data. You’ll extend its `metadata` when you need to store something intended to be added to a specific `Account` or `Domain` instance.

Reviewer: ```suggestion
You must use `metadata` when you don’t want sequences of records, for example, extending user data. You can extend its `metadata` when you need to store something intended to be added to a specific `Account` or `Domain` instance.
```

> You must use a `Store` asset if the data exists in the blockchain and a default number type is not applicable. It lets you record data sequences and works as a dictionary, where the keys are string, while the values are  `Value` type instance that assumes many types starting with a `String` and finishing with `Ipv6Addr`.

Reviewer 2: ```suggestion
You must use a `Store` asset if the data exists in the blockchain and a default number type is not applicable. It lets you record data sequences and works as a dictionary, where the keys are strings, while the values are  `Value` type instances that assume many types from `String` to `Ipv6Addr`.
```

Author: Thanks!

Let’s discuss some use cases to make it more clear.

> Let’s say you have a clinic. Here, generic user data, such as birthdays, can be added to the `metadata`. The `Store` asset instances record the appointments, treatments suggested by the doctors, and medical test results.

Reviewer: ```suggestion
Let’s say, for example, you have a clinic. Here, generic user data, such as birthdays, can be added to `metadata`. The `Store` asset instances record the appointments, treatments suggested by the doctors, and medical test results.
```
